llvm/test/MC/RISCV/corev/simd-all-extensions.s

```diff
@@ -944,8 +944,8 @@ cv.srl.sc.b s0, s1, s2
 # CHECK-ERROR: error: instruction requires the following: 'Xcvsimd' (SIMD ALU)
 # CHECK-UNKNOWN: 7b d4 24 41 <unknown>
 
-cv.srl.sci.h t0, t1, 32
-# CHECK-INSTR: cv.srl.sci.h t0, t1, 32
+cv.srl.sci.h t0, t1, -32
+# CHECK-INSTR: cv.srl.sci.h t0, t1, -32
 # CHECK-ENCODING: [0xfb,0x62,0x03,0x41] 
```

Comment:
I am not convinced that LLVM is wrong. The specification of this instruction says the shift is unsigned. This may be a mistake in the GNU assembler